config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
-  - manager.yaml
\ No newline at end of file
+- manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
Please restore also this file.

Reply:
@m1o1, could you please also sign your PR?

Reply:
done